src/Entity/User.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Security\Core\User\UserInterface;
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+
+/**
+ * @ORM\Entity
+ * @UniqueEntity(fields="email", message="Email already taken")
+ * @UniqueEntity(fields="username", message="Username already taken")
+ */
+class User implements UserInterface
```

Comment:
Il faut implémenter l'interface \Serializable.
Cela permet a symfony de vérifier si l'utilisateur a été modifié. Tu trouvera des explications ici [https://symfony.com/doc/current/security/entity_provider.html#what-do-the-serialize-and-unserialize-methods-do](url)
Les deux méthodes que tu vas devoir écriresuite a l'ajout de cette interface permette de savoir comment est stocké l'utilisateur en session

Reply:
C'est noté